app/src/main/java/com/uc/mymovies/ListAdapter.kt

```diff
@@ -30,18 +30,18 @@ class ListAdapter(private val context: Context, var movies: ArrayList<Movie>):Ba
             viewHolder = view.tag as ViewHolder
         }
 
-        viewHolder.name?.text = movies[position].title;
-        var imageURL = movies[position].image;
-        Picasso.get().load(imageURL).into(viewHolder.image);
```

Comment:
A hard habit to break.

Reply:
I do a lot of reactjs and often always leave them out. Definitely a habit I need to break!